src/lib.rs

```diff
@@ -9,7 +9,8 @@
 
 // From rustc.
 extern crate arena;
-#[macro_use] extern crate rustc;
+#[macro_use]
+extern crate rustc;
```

Comment:
Good point.

I would put empty lines around the `#[macro_use] extern crate` line though. And re-order the imports so that the `#[macro_use]` ones come first.